### PR TITLE
fix: light mode leftovers

### LINKED ISSUE #3972: Light mode leftovers
Collection select at /create
image

Filtering collections at /explore:
image

On hover overview of collection at /explore:
image

Collection history select:
image

Item Detail history select:
image

Profile created sorting:
image

Profile history sorting:
image

Profile sold sorting:
image

### PR DESCRIPTION
**Pull request by alohaw0**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [X] PR closes #3972
- [ ] Requires deployment <rubick/magick_issue>
- [X] fix: light mode leftovers

### Before submitting Pull Request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality
- [X] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [X] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer?target=HL37EEfDA8Bfn5E7sHxuNn1xHuT1wmeQKaQfZV3eTq1XymN&usdamount=100&donation=true)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [X] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
图片
图片
图片
图片
etc...

### CODE REVIEW

**Comment by petersopko** [comment]
nice, wfm

**Review by petersopko — APPROVED** [review]

File: styles/themes/_light.scss [hunk 1]
```diff
@@ -243,6 +247,7 @@ html.light-mode {
 		color: $text-light-theme;
 	}
 
+	.card .card-content,
```
Comment (roiLeo) [inline styles/themes/_light.scss:250]: Capture d’écran 2022-09-19 à 11 57 20 AM

What are you trying to fix here?

**Review by roiLeo — CHANGES_REQUESTED** [review]
Small note:
`.card .card-content` css rule is going to overide all card content design from app

**Comment by petersopko** [comment]
@alohaw0 let's please adjust the changes to only reflect what needs to be done based on the issue :)

**Comment by alohaw0** [comment]
> @alohaw0 let's please adjust the changes to only reflect what needs to be done based on the issue :)

I noticed that the .nft-card is the only class works to cards in that case. 
So I changed the `.card` to `.nft-card` in the newer commit.

@petersopko @roiLeo PTAL

**Review by roiLeo — APPROVED** [review]
✅ code looks good

**Comment by petersopko** [comment]
wfm 🚀

**Comment by petersopko** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 39.88 USD/KSM ~ 0.752 $KSM
🧗 [HL37EEfDA8Bfn5E7sHxuNn1xHuT1wmeQKaQfZV3eTq1XymN](https://kusama.subscan.io/account/HL37EEfDA8Bfn5E7sHxuNn1xHuT1wmeQKaQfZV3eTq1XymN)
🔗 [0x2a6b5bd3878d33cc35faaf1491c8c8f4991615c8a8658549e1aa6a3d6817edbe](https://kusama.subscan.io/extrinsic/0x2a6b5bd3878d33cc35faaf1491c8c8f4991615c8a8658549e1aa6a3d6817edbe)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**